clv/data_access.py

```diff
@@ -81,21 +81,22 @@ def query_data_source(self):
             try:
                 for sep in [',', ';', ':']:
 
-                    self.data = pd.read_csv(filepath_or_buffer=join(conf('data_main_path'), self.data_query_path),
+                    self.data = pd.read_csv(filepath_or_buffer=self.data_query_path,
```

Comment:
it is only for a connection to DB. 
There is no need 'data_main_path' variable from configs.py in order to read a file from a folder.